backend/src/services/streamStore.ts

```
@@ -83,6 +83,7 @@ function rowToRecord(row: StreamRow): StreamRecord {
     completedAt: row.completed_at ?? undefined,
     refundedAmount: row.refunded_amount ?? undefined,
     pausedAt: row.paused_at ?? undefined,
+    pausedDuration: row.paused_duration ?? 0,
```

Biome (2.4.13)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)